test/TEST-30-ONCLOCKCHANGE/test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+set -e
+TEST_DESCRIPTION="test OnClockChange= + OnTimezoneChange="
+TEST_NO_NSPAWN=1
+
+. $TEST_BASE_DIR/test-functions
+
+test_setup() {
+    create_empty_image
+    mkdir -p $TESTDIR/root
+    mount ${LOOPDEV}p1 $TESTDIR/root
+
+    (
+        LOG_LEVEL=5
+        eval $(udevadm info --export --query=env --name=${LOOPDEV}p2)
+
+        inst_any /usr/share/zoneinfo/Europe/Kiev
```

Comment:
I guess `/usr/share/zoneinfo/Europe/Berlin` should be there as well to avoid `Failed to set time zone: Invalid time zone 'Europe/Berlin`

Reply:
ouch! force pushed a new version with this fixed.